src/commands/info.js

```diff
@@ -1,16 +1,34 @@
 // @flow
 import * as options from '../utils/options';
 import { BoltError } from '../utils/errors';
+import * as yarn from '../utils/yarn';
 
-export type InfoOptions = {};
+export type InfoOptions = {|
+  cwd?: string,
+  args: options.Args
+|};
 
 export function toInfoOptions(
   args: options.Args,
   flags: options.Flags
 ): InfoOptions {
-  return {};
+  const depsArgs = Array.prototype.concat([], args);
+  Object.keys(flags).forEach(key => {
+    if (flags[key] === true) {
+      depsArgs.push(`--${key}`);
```

Comment:
We shouldn't convert flags like this. We should explicitly support individual flags

Reply:
> We should explicitly support individual flags

I didn't understand this.  I've seen the same kind of implementation in the project https://github.com/boltpkg/bolt/blob/master/src/utils/yarn.js#L16.

Reply:
That's a very explicit conversion. I'm saying we shouldn't accept arbitrary flags and pass them through